test/test-config.sh

```diff
@@ -162,7 +162,7 @@ fi
 # EndpointSlice is disabled because of https://github.com/kubernetes/kubernetes/issues/91287 (Kubernetes
 # < 1.19) and because there were random connection failures to node ports during sanity
 # testing (Kubernetes 1.19.0)
-: ${TEST_FEATURE_GATES:=CSINodeInfo=true,CSIDriverRegistry=true,CSIBlockVolume=true,CSIInlineVolume=true,\
+: ${TEST_FEATURE_GATES:=\
```

Comment:
Removal of GA features is fine. That's what the warning is about.

I'm less sure about beta features. It is possible that support for them gets removed in a future release. When we explicitly enable one that was removed, we get an unambiguous error when setting up the new Kubernetes release. When we don't enable them, the cluster comes up, but then won't work as expected (i.e. some test failure during E2E testing).